### ISSUE TITLE
Fehler beim Abrufen der Account-Informationen

### ISSUE THREAD

**Issue by spleijers** [beaTejakulator/tibber_reading_uploader#14]
```
==================================================================================================
################ Starting Tibber Uploader service at Fri Aug  2 01:00:06 CEST 2024 ################
==================================================================================================
2024-08-02 01:00:27 - Token erfolgreich abgerufen.
2024-08-02 01:00:27 - Upload-Prozess erfolgreich gestartet.
2024-08-02 01:00:28 - Fehler beim Abrufen der Account-Informationen: 'NoneType' object has no attribute 'get'
Nächste Übermittlung um 01:00 Uhr
```
Hello, im using this uploader to get some usage info uploaded to Tibber. Unfortunately, i get the error; Fehler beim Abrufen der Account-Informationen

Any idea how to fix this?

**Comment by MarvinEis** [comment]
@sanderpleijers 

Has the uploader ever worked for you before?
I have adapted the code again for better analysis. Please try the update.

**Comment by spleijers** [comment]
@beaTejakulator; No it never has worked before. With the newer version the error seems to be different; `Fehler beim Abrufen der Account-Informationen: Konnte keine gültige meter_id oder register_id finden.`

I would very much like to use this addon, because the Tibber Pulse meter does not seem work with my older (Iskra MT382) power meter.

**Comment by MarvinEis** [comment]
Can you use the API in general? 

https://developer.tibber.com/explorer

**Comment by spleijers** [comment]
Yes, the API works but i saw a wierd thing there that was not good; i did not have any 'usage' (the app also showed nothing). I contacted Tibber support and they fixed that for me. Even more, they also fixed my Tibber Pulse! So the need for this addon is gone with that. If you want me to test it however, i will be available to help. The warning seems to be the same. 

Btw, im a Dutch Tibber user. Maybe there is a small difference here in the API?
